addons/advanced_throwing/functions/fnc_drawThrowable.sqf

```diff
@@ -137,7 +137,7 @@ if (ACE_player getVariable [QGVAR(dropMode), false]) then {
     _posFin = _posFin vectorAdd (AGLToASL (positionCameraToWorld [_leanCoef, 0, ACE_player getVariable [QGVAR(dropDistance), DROP_DISTANCE_DEFAULT]]));
 
     // Even vanilla throwables go through glass, only "GEOM" LOD will stop it but that will also stop it when there is no glass in a window
-    if (lineIntersects [_posCameraWorld, _posFin vectorDiff _posCameraWorld]) then {
+    if !((lineIntersectsSurfaces [_posCameraWorld, _posFin vectorDiff _posCameraWorld]) isEqualTo []) then {
```

Comment:
This has to be tested with all kinds of glass too. Some CUP buildings for example don't let anything through glass (A2 config).

I am more worried about letting grenades be dropped inside the glass without glass being broken.

Reply:
Reverted for the moment